ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -100,4 +108,37 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
       session_id
     );
   }
+
+  /**
+   * @param {object} event
+   */
+  @action
+  async handleInput(event) {
```

Comment:
This will also need to be debounced

Reply:
This will be address in another PR, ticket [here](https://hashicorp.atlassian.net/browse/ICU-11851)